https://issues.redhat.com/browse/ACM-13627 for 2.12 (#6834)
open-cluster-management/rhacm-docs: release_notes/release_notes.adoc

* xref:../release_notes/observability_support.adoc#observability-support[Observability support]

If you experience issues with one of the currently supported releases, or the product documentation, go to link:https://www.redhat.com/en/services/support[Red Hat Support] where you can troubleshoot, view Knowledgebase articles, connect with the Support Team, or open a case. You must log in with your credentials.
You can also learn more about the Customer Portal documentation at link:https://access.redhat.com/articles/33844[Red Hat Customer Portal FAQ].

> The documentation team links to the earliest supported {ocp} versions that are tested by Quality Engineering, but later versions are supported. See the link:https://access.redhat.com/support/policy/updates/advanced-cluster-management[Lifecycle & Update Policies of for {acm}] for more details about the support version strategy.

Reviewer: We don't want to point to the humans doing the work here; in a more informal doc we can. 

I would say "The documentation references the earliest ........., unless the component in the documentation is created and tested with only a specific version of {ocp-short}. For full support information, see (add support matrix first) and (add lifecycle).

This should go in both ACM and MCE release notes introductions.

> The documentation team links to the earliest supported {ocp} versions that are tested by Quality Engineering, but later versions are supported. See the link:https://access.redhat.com/support/policy/updates/advanced-cluster-management[Lifecycle & Update Policies of for {acm}] for more details about the support version strategy.

Author: ```suggestion
The documentation references the earliest supported {ocp} versions, unless the component in the documentation is created and tested with only a specific version of {ocp-short}. For full support information, see the link:https://access.redhat.com/articles/7073065[Support matrix] and the link:https://access.redhat.com/support/policy/updates/advanced-cluster-management[Lifecycle & Update Policies for {acm}].
```

> The documentation references the earliest supported {ocp} versions, unless the component in the documentation is created and tested with only a specific version of {ocp-short}. For full support information, see the link:https://access.redhat.com/articles/7073065[Support matrix] and the link:https://access.redhat.com/support/policy/updates/advanced-cluster-management[Lifecycle & Update Policies for {acm}].

Author: ```suggestion
The documentation references the earliest supported {ocp} versions, unless the component in the documentation is created and tested with only a specific version of {ocp-short}. For full support information, see the link:https://access.redhat.com/articles/7073065[Support matrix] and the link:https://access.redhat.com/support/policy/updates/advanced-cluster-management[Lifecycle and update policies for {acm}].
```

> The documentation references the earliest supported {ocp} versions, unless the component in the documentation is created and tested with only a specific version of {ocp-short}. For full support information, see the link:https://access.redhat.com/articles/7073065[Support matrix] and the link:https://access.redhat.com/support/policy/updates/advanced-cluster-management[Lifecycle and update policies for {acm}].

Author: ```suggestion
The documentation references the earliest supported {ocp} versions, unless the component in the documentation is created and tested with only a specific version of {ocp-short}.

 For full support information, see the link:https://access.redhat.com/articles/7073065[Support matrix] and the link:https://access.redhat.com/support/policy/updates/advanced-cluster-management[Lifecycle and update policies for {acm}].
```
